docs/advanced/html.md

```diff
@@ -105,9 +105,37 @@ html:
 
 :::{seealso}
 - For more about Google Analytics, see [the Google Analytics documentation](https://analytics.google.com/analytics/web/) for more information.
-- To use **other analytics services**, like [Plausible analytics](https://plausible.io/), you should link the relevant libraries via the instructions in [](custom-assets).
 :::
 
+
+## Use Plausible Analytics
+
+[Plausible Analytics](https://plausible.io) is a lightweight, open source, [privacy-focused](https://plausible.io/privacy-focused-web-analytics) analytics service that can be used as a more ethical alternative (or, in addition to) to Google Analytics.
+The requirement to use Plausible.io is to add a snippet into the `<head>` section of every html page:
+
+```
+<script defer data-domain="YOUR-DATA-DOMAIN" src="https://plausible.io/js/plausible.js"></script>
+```
+
+It is not yet possible in JupyterBook to directly inject `html` code into the site `<head>`, but it is possible to add [](custom-assets).
+te
+To use Plausible Analytics in your JupyterBook, simply add this code into an arbitrarily named `.js` file in the `_static` directory (create the directory in the root of your book if it does not already exist):
```

Comment:
```suggestion
To use Plausible Analytics in your JupyterBook, add this code into an arbitrarily-named `.js` file in the `_static` directory (create the directory in the root of your book if it does not already exist):
```

Reply:
we should try to avoid uses of words like `simply`, `easy`, etc since it is highly dependent on the person

Reply:
ha! yes, absolutely right!